build.gradle.kts

```diff
@@ -59,4 +67,5 @@ dependencies {
 
 tasks.getByName<Test>("test") {
     useJUnitPlatform()
+    jvmArgs("--enable-preview")
```

Comment:
Maybe you can add the --enable-preview argument in the readme tutorial for running the node.